### ISSUE TITLE
Missing primitive type should default to 3

### ISSUE THREAD

**Issue by sebjf** [3drepo/3drepo.io#3948]
### Description

When loading the sample Lego house model in a Streaming viewer, the model fails to load with an error reporting the primitive type is incomplete.

```
Bundle carmen.844239a0-00c6-11ec-8d97-41a278fb55fd.89d5580a-3224-4e50-bbab-89d855c320e0 does not have a primitive type. It is likely the metadata is incomplete. {"_id":"89d5580a-3224-4e50-bbab-89d855c320e0","nVertices":0,"nFaces":6019,"nUVChannels":1,"primitive":0,"min":[0.0,0.0,-14380.0],"max":[13580.0,3200.0,0.0]}
```

- This is happening in: staging
- This requires a hotfix to production: no 

### Steps to replicate
- Open the Lego House in a Streaming Viewer

### Current Behaviour
- Model fails to load with the above error

### Expected Behaviour
- Model loads OK

**Comment by sebjf** [comment]
This is because the `Scene.getMeshInfo` sets the primitive field to 0 if it is missing. This field should default to 3, like `getMeshById`, for backwards compatibility with models imported before lines were supported.

**Comment by carmenfan** [comment]
Merged into #3910
